packages/react/src/components/Button/index.js

```diff
@@ -21,6 +29,10 @@ Button.propTypes = {
   as: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
   className: PropTypes.string,
   disabled: PropTypes.bool,
+  forwardedRef: PropTypes.oneOfType([
```

Comment:
In order to DRY, I guess this propType should be exported and imported similarly to how stubs are done

Reply:
Done 👍🏼